Makefile

```diff
@@ -31,6 +31,10 @@ clean:
 distclean: clean
 	find vendor/ -maxdepth 1 -mindepth 1 -not -name vendor.json -exec rm {} -rf \;
 	rm -f manifest/*.yaml
+	rm -f .glide.status
+
+checksync: .glide.status
+	test -d vendor || make sync
```

Comment:
For the current implementation if glide.yaml needs to be run, "glide update" gets called right before this check. Without this check "glide install" will blow away and re-install a freshly laid vendor folder.